File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/importering/autoimport/statistikk/utils/SykefraværsstatistikkIntegrasjonUtils.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.importering.autoimport.statistikk.utils;

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.importering.Sykefraværsstatistikk;

import java.util.List;

public interface SykefraværsstatistikkIntegrasjonUtils {

    DeleteSykefraværsstatistikkFunction getDeleteFunction();

    BatchCreateSykefraværsstatistikkFunction getBatchCreateFunction(List<? extends Sykefraværsstatistikk> list);

}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/infrastruktur/database/SykefraværsstatistikkIntegrasjonUtils.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.infrastruktur.database

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.fellesdomene.Sykefraværsstatistikk

interface SykefraværsstatistikkIntegrasjonUtils {
    fun getDeleteFunction(): DeleteSykefraværsstatistikkFunction
    fun getBatchCreateFunction(
        statistikk: List<Sykefraværsstatistikk>
    ): BatchCreateSykefraværsstatistikkFunction
}
